### PR TITLE
Remove snow

### PR DESCRIPTION
**Pull request by Maijin**
**Your checklist for this pull request**
- [x] I've read the [guidelines for contributing](https://github.com/rizinorg/rizin/blob/master/DEVELOPERS.md) to this repository
- [x] I made sure to follow the project's [coding style](https://github.com/rizinorg/rizin/blob/master/DEVELOPERS.md#code-style)
- [x] I've added tests that prove my fix is effective or that my feature works (if possible)
- [x] I've updated the documentation and the [rizin book](https://github.com/rizinorg/rizinbook) with the relevant information (if needed)

**Detailed description**

Remove snow.

### CODE REVIEW

File: librz/core/panels.c [hunk 1]
```diff
@@ -1720,30 +1715,6 @@ bool __handle_window_mode(RzCore *core, const int key) {
 		free (cmd);
 	}
 		break;
-	case 'h':
-		(void)__move_to_direction (core, LEFT);
```
Comment (ret2libc) [inline librz/core/panels.c:1724]: do you really need to remove this? it doesn't seem to be used only when snow/fun is used, is it?

Comment (Maijin) [reply librz/core/panels.c:1724]: It is only used in snow

Comment (Maijin) [reply librz/core/panels.c:1724]: oh wait you're right

Comment (Maijin) [reply librz/core/panels.c:1724]: woups, fixed

**Comment by Maijin** [comment]
Broken test looks unrelated (sdb)

**Review by XVilka — APPROVED** [review]
These tests fail randomly from time to time, probably worth investigating. But they are indeed unrelated.

**Review by wargio — APPROVED** [review]

**Pull request merged**